src/css/bookstore.css

```diff
@@ -279,6 +279,10 @@ body {
   opacity: 1 !important;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230429417-60682db5-8639-4f69-b6a7-83fc2fdf072e.png)

- [ ] First, kindly remove the screenshots section from your readme file because you don't have any screenshots in that section. So kindly remove this section to make your readme descriptive 😊

Reply:
I have been penalized several times for not including that section, and N/A was advised as placeholder